src/library_browser.js

```diff
@@ -881,7 +882,7 @@ var LibraryBrowser = {
   $getPreloadedImageData: (path, w, h) => {
     path = PATH_FS.resolve(path);
 
-    var canvas = /** @type {HTMLCanvasElement} */(preloadedImages[path]);
+    var canvas = /** @type {HTMLCanvasElement} */(Browser.preloadedImages[path]);
```

Comment:
Missing dependency on $Browser?

Reply:
All symbols in this file implicitly depend on `$Browser` thanks to the `autoAddDeps(LibraryBrowser, '$Browser');` line at the bottom.